ParabolInc/parabol
# feat(sprint-poker): Add GitLab issue

Fix #6033 

### To test

- Integrate with GitLab (you may need to add the feature flag if you can't see GitLab in the integration provider row)
- Make sure you're part of the GitLab team Parabol1
- Create a Sprint Poker meeting, go to the Scope phase, and see your GitLab issues
- Click the FAB button, select a project, enter a title, and see that the issue has been created
